Logging/Logging.psd1

```diff
@@ -12,7 +12,7 @@
 RootModule = 'Logging.psm1'
 
 # Version number of this module.
-ModuleVersion = '2.6.13'
+ModuleVersion = '2.6.15'
```

Comment:
Please do not update `ModuleVersion` as it is updated by the build script using gitversion semver.

Reply:
Oh I see what happens here. I pulled your master into my fork, which brought these updates to the manifest file. I didn't modify these by myself. Same with  the changelog file below

Reply:
Ah ok, so no problem